src_tests/test_simulation_strategies.py

```diff
@@ -6,21 +6,32 @@
 stuff with only minor differences (mostly in experiments setup). 
 """
 
+from typing import List
 import unittest
 from copy import deepcopy
 from math import factorial
+from scipy.special import binom
 
 from numpy import array, asarray, average
 from numpy.random import randint
 
 from src.Boson_Sampling_Utilities import calculate_number_of_possible_n_particle_m_mode_output_states
 from src.LossyBosonSamplingExactDistributionCalculators import BosonSamplingExactDistributionCalculator, \
-    BosonSamplingExperimentConfiguration, BosonSamplingWithFixedLossesExactDistributionCalculator
+    BosonSamplingExperimentConfiguration, BosonSamplingWithFixedLossesExactDistributionCalculator, BosonSamplingWithUniformLossesExactDistributionCalculator
```

Comment:
Nitpick: `tuple import` would be nice in this case (can be enabled in PyCharm).

Reply:
Is this just putting longer imports in the parentheses? Something like asked here:
https://stackoverflow.com/questions/22828997/python-import-as-tuple ?

Reply:
Yes, but it would be even nicer with one line per imported element.

Reply:
Done.